liveRecorder/ts_to_mp4_convertor/ts_to_mp4_convertor.xcodeproj/project.pbxproj

```diff
@@ -0,0 +1,319 @@
+// !$*UTF8*$!
```

Comment:
need to add make file for linux + build script
move the binary to the bin folder darwin and linux